docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,189 @@
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](/contributing/gatsby-style-guide/)
+
+#### Code
+
+For [PRs that add code](/contributing/code-contributions/) (whether a feature or fix), we look for the following:
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour. This can be a combination of unit tests for the specific package, snapshot tests, and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this.
+- Code Quality — focus on reasonable changes that will likely improve code maintenance, comprehension, or correctness. Stylistic changes are typically linted for by Prettier. Don't nitpick.
+- Documentation in the package's README if you're adding something
+
+#### Starters or Site Showcase
+
+For PRs that add a site or a starter to the showcase, we ought to check:
+
+- Check if the site or starter is built with Gatsby
+- Links — check if the links are working and accessible
+- Tags — ensure the tags match existing tags
+- Featured Status — new sites should not be marked as featured. Featured sites are occasionally updated by a member of the Gatsby team.
+
+#### Blog posts
+
+For PRs that add a blog post, we ought to check:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](/contributing/gatsby-style-guide/)
+- Subject matter — blog posts should not be purely promotional, spammy, or inappropriate. An author should check with a member of the Gatsby team that their post is appropriate for the blog before creating their PR.
+- Time Sensitivity — blog posts are more time dependent than docs, especially since they get buried after more posts are published. If something is continually relevant and more of a general how-to, it should probably go in the [Reference Guides](/docs/guides/) section of the docs.
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds for [gatsbyjs.org](https://www.gatsbyjs.org).
+
+We want all of these checks to pass. While it's okay to review a work in progress PR with some failed checks, a PR is only ready to ship when all the tests have passed.
+
+Let's go over some common failure cases and how to fix them:
+
+### Linting
+
+We lint all code and documentation for consistency. You might find that your PR fails on the linting check.
+
+If this is your PR and you have the code checked out on your machine, you can run:
+
+> 💡 npm run format
```

Comment:
While fun, I wonder if the lightbulb will confuse people at all. Perhaps it should be marked up as a bash code snippet?

```bash
npm run format
```